src/network/core/packet.h

```diff
@@ -40,47 +42,49 @@ typedef uint8  PacketType; ///< Identifier for the packet
 struct Packet {
 	/** The next packet. Used for queueing packets before sending. */
 	Packet *next;
-	/**
-	 * The size of the whole packet for received packets. For packets
-	 * that will be sent, the value is filled in just before the
-	 * actual transmission.
-	 */
-	PacketSize size;
+private:
```

Comment:
maybe move `Packet *next` to after `public` too? Just to make clear it is public.

Reply:
Alternatively, and fits with the whole encapsulation... make it private.